spec/fixtures/custom_plugins/kong/plugins/dbless-pagination-test/handler.lua

```diff
@@ -0,0 +1,22 @@
+-- The test case 04-client_ipc_spec.lua will load this plugin and check its
+-- generated error logs.
+
+local LmdbPaginationTestHandler = {
+  VERSION = "1.0",
+  PRIORITY = 1000,
+}
+
+
+local function test()
+  local db = kong.db
+
+  assert(db.routes.pagination.max_page_size == 2048)
+end
+
+
+function LmdbPaginationTestHandler:init_worker()
```

Comment:
Should we send a request instead? `timer.at(0)` could make the test flaky.

Reply:
Currently `test()` only includes one `assert`, I think it is ok, more tests will be in access handler.

Reply:
Another question, why not use https://docs.konghq.com/hub/kong-inc/post-function/ instead of writing a plugin? We usually only need to write a new plugin if custom DAO is needed.

Reply:
done.